app/assets/stylesheets/includes/forms.scss

```diff
@@ -36,6 +36,20 @@ section.sign-up {
   .btn-link {
     color: $white
   }
+  h1, h2, h3, h4, h5 {
+    color: $white;
+    padding: 6px 12px;
```

Comment:
I usually prefer to use margin instead of padding for spacing titles. In this case I would guess margin-top 30px (20px default + 10px that you were already doing with padding) and margin-bottom 15px. That's my personal preference. What do you think?